autogpt_platform/frontend/src/components/molecules/CookieConsent/index.ts

```diff
@@ -0,0 +1,2 @@
+export { CookieConsentBanner } from "./CookieConsentBanner";
+export { CookieSettingsModal } from "./CookieSettingsModal";
```

Comment:
Let's [avoid index files](https://github.com/Significant-Gravitas/AutoGPT/blob/master/autogpt_platform/frontend/CONTRIBUTING.md#guidelines), it's mostly boilerplate ( _for imports paths to look good_ ) and they can slow down bundlers.